src/test/java/in/zapr/druid/druidry/query/aggregation/GroupByTest.java

```diff
@@ -71,56 +71,56 @@ public void init() {
     @Test
     public void testSampleQuery() throws JsonProcessingException, JSONException {
         String expectedJsonAsString = "{\n" +
-                "  \"queryType\": \"groupBy\",\n" +
```

Comment:
Could you remove this(and at few other places) formatting git diff as it unnecessarily rewrites git history. If you use style guide I mentioned in conversion comment it should not happen. Or if it's because of your local tab/space setting could you reformat to what is there in existing commit so there's no formatting git diff.

Reply:
Actually this was suggested by you as step 2 above. I formatted code using Google Style Guide. Reverting as you requested.

Reply:
Sorry for that, I think code style use tabs and tabs are again considered according to local setting of tabs. But thank you for reverting formatting changes.